internal/cmd/server/start_resource_server.go

```diff
@@ -502,9 +512,6 @@ func (c *ResourceServerCommand) generateSearchApiUrl(backendURL string) (string,
 	// Split URL address
 	hostArr := strings.Split(u.Host, ".")
 
-	// Replace with search API prefix
-	hostArr[0] = searchApiUrlPrefix
```

Comment:
Will it work when running the resource-server manually (i.e. without the operator)?
I mean, IIUC, after this change the search-api prefix is added only using the operator.
We probably need to add another flag the specify the prefix or the namespace? e.g. --search-api-namespace

Reply:
The search-api is the backendURL of the resource-server which would be provided in full when run manually.
This is how I've been running the resource server:
`./oran-o2ims start resource-server --log-level=debug --log-file=stdout --api-listener-address=localhost:8002 --cloud-id=123 --backend-url="${BACKEND_URL}" --backend-token-file=./token
`
where BACKEND_URL is `https://search-api-open-cluster-management.apps.lab.karmalabs.corp`. There is no need to override with the search-api prefix.
@danielerez , do you agree?

When running with the operator, the operator will try to dynamically construct all the search-api URL.

Reply:
Oh, so maybe we should just rename the BACKEND_URL in [launch.json](https://github.com/openshift-kni/oran-o2ims/blob/34985222663b8261bdc7b9217f19dc4fac391bcb/.vscode/launch.json#L54) to something like SEARCH_API_URL.
I.e. the current code was added in order to avoid changing the url when debugging a different server. It's not crucial of course, but should be easier when running the server locally.
Any way, can you please update the [readme](https://github.com/openshift-kni/oran-o2ims/blob/main/README.md#resource-server) with details about the behaviour (to note that 'search-api' prefix is needed in the url).

Reply:
I left everything with BACKEND_URL to keep it consistent.  Maybe I can add some extra explanation in the CRD about the resource server requiring the search API.

Reply:
Yeah, sounds good. Also in the readme file to emphasis that the prefix is required.